src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -64,6 +66,8 @@
 
     protected InetSocketAddress clientPortAddress;
     protected InetSocketAddress secureClientPortAddress;
+    protected boolean sslQuorum = false;
+    protected boolean shouldUsePortUnification = false;
```

Comment:
these options are not available as system properties and can only be set in the config file.